src/main/java/carracing/domain/RacingResult.java

```diff
@@ -0,0 +1,40 @@
+package carracing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingResult {
+    private final List<RacingCar> racingCars;
+
+    public RacingResult(List<RacingCar> racingCars) {
+        this.racingCars = racingCars;
+    }
+d
+    public List<RacingCar> getRacingCars() {
+        return racingCars;
+    }
+
+    public List<String> findWinners() {
+        return findNameByPosition(findMaxPosition());
+    }
+
+    private List<String> findNameByPosition(Position maxPosition) {
+        List<String> winners = new ArrayList<>();
+        for (RacingCar racingCar : racingCars) {
+            if (racingCar.isGreaterThanPosition(maxPosition)) {
+                winners.add(racingCar.getName());
+            }
+        }
+        return winners;
+    }
+
+    private Position findMaxPosition() {
+        Position maxPosition = new Position();
+        for (RacingCar racingCar : racingCars) {
+            if (racingCar.isGreaterThanPosition(maxPosition)) {
+                maxPosition = racingCar.getPosition();
+            }
```

Comment:
```suggestion
            maxPosition = racingCar.max(maxPosition);
```
객체의 값을 꺼내지 말고 메시지를 보낸다.